intel_extension_for_transformers/llm/runtime/graph/application/main_pybind.cpp

```diff
@@ -379,5 +403,6 @@ PYBIND11_MODULE(polyglot_cpp, m)
                   py::arg("weight_dtype") = "int4", py::arg("alg") = "sym", py::arg("group_size") = 32,
                   py::arg("scale_dtype") = "fp32", py::arg("compute_dtype") = "ggml", py::arg("use_ggml") = false)
       .def("is_token_end", &Model::is_token_end)
+      .def("reset_token_end", &Model::reset_token_end)
```

Comment:
so we can reset end token to other logit id ?

Reply:
just reset status for next prompt